app/controllers/trending_controller.rb

```diff
@@ -18,7 +18,7 @@ def top_pages
   end
 
   def most_frequent_phrases
-    Phrase.most_frequent(filter_start_date, filter_end_date)
+    Phrase.most_frequent(filter_start_date, filter_end_date).take(10)
```

Comment:
We're (probably) always going to want to limit the number of phrases, would it make sense to make this a parameter (ie `Phrase.most_frequent(filter_start_date, filter_end_date, 10)` and add a `limit` to the query. Did a little bit of testing and `take` is included in the sql query so neither is more efficient

Reply:
Potentially, though at the moment I'd prefer to keep the models as uncoupled to the consumers as possible. If we add a parameter for `limit` then we need to default it to something which means we're always limiting. By not having a limit then the consumers are in complete control of what to do with the data i.e. paginate over it (this goes wider than this model).

Reply:
good point